### ISSUE TITLE
Some deprecations with Symfony 5.4 & PHP 8.1

### ISSUE THREAD

**Issue by Huluti** [mhujer/BreadcrumbsBundle#30]
Hi! Just a quick issue for some deprecations we have on a website with Symfony 5.4 & PHP 8.1. Here's the list:

```
Return type of WhiteOctober\BreadcrumbsBundle\Model\Breadcrumbs::current($namespace = self::DEFAULT_NAMESPACE) should either be compatible with Iterator::current(): mixed, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice

Return type of WhiteOctober\BreadcrumbsBundle\Model\Breadcrumbs::next($namespace = self::DEFAULT_NAMESPACE) should either be compatible with Iterator::next(): void, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice

Return type of WhiteOctober\BreadcrumbsBundle\Model\Breadcrumbs::key($namespace = self::DEFAULT_NAMESPACE) should either be compatible with Iterator::key(): mixed, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice

Return type of WhiteOctober\BreadcrumbsBundle\Model\Breadcrumbs::rewind($namespace = self::DEFAULT_NAMESPACE) should either be compatible with Iterator::rewind(): void, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice

Return type of WhiteOctober\BreadcrumbsBundle\Model\Breadcrumbs::offsetGet($offset, $namespace = self::DEFAULT_NAMESPACE) should either be compatible with ArrayAccess::offsetGet(mixed $offset): mixed, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice

Deprecated: Return type of WhiteOctober\BreadcrumbsBundle\Model\Breadcrumbs::current($namespace = self::DEFAULT_NAMESPACE) should either be compatible with Iterator::current(): mixed, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice

Deprecated: Return type of WhiteOctober\BreadcrumbsBundle\Model\Breadcrumbs::next($namespace = self::DEFAULT_NAMESPACE) should either be compatible with Iterator::next(): void, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice

Deprecated: Return type of WhiteOctober\BreadcrumbsBundle\Model\Breadcrumbs::key($namespace = self::DEFAULT_NAMESPACE) should either be compatible with Iterator::key(): mixed, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice

Deprecated: Return type of WhiteOctober\BreadcrumbsBundle\Model\Breadcrumbs::rewind($namespace = self::DEFAULT_NAMESPACE) should either be compatible with Iterator::rewind(): void, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice

Deprecated: Return type of  WhiteOctober\BreadcrumbsBundle\Model\Breadcrumbs::offsetGet($offset,  $namespace = self::DEFAULT_NAMESPACE) should either be compatible with  ArrayAccess::offsetGet(mixed $offset): mixed, or the  #[\ReturnTypeWillChange] attribute should be used to temporarily  suppress the notice
```

Thanks for your work :)

**Comment by mhujer** [comment]
@Huluti Hi! It should be fixed in #31. I've tagged it as 1.5.8, can you please give it a try?

**Comment by Huluti** [comment]
Hi! This works well, no more deprecations. Thank you very much for your reactivity, very appreciated :)
